index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello Microverse</title>
+    <!-- External CSS -->
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="test">
+    <h1>Hello Microverse </h1>
+    <p>This is a project which is a good practice to understand Linters</p>
+    </div>
+</body>
+</html>
```

Comment:
- [ ] Kindly write a more descriptive PR because it's the first thing that will provide information about the changes you introduced. 🤝️
You can refer to this [material](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) to get more information on how to write a descriptive PR.

- [ ] Kindly update your README file with the template provided by Microverse. A README file will provide an essential summary of the project. 🥸️
You can get the recommended README template [here](https://github.com/microverseinc/readme-template).

- [ ] [OPTIONAL] Nice work capitalizing your commit messages. 👍️
I think it will be better if you write them in the imperative tense as it is suggested by Microverse [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)

Reply:
do I need to change p tag

Reply:
@mukund816, No need to change the `p` tag.